packages/web-app-files/src/views/spaces/Project.vue

```diff
@@ -9,6 +9,7 @@
           :cancel="closeReadmeContentModal"
           :space="space"
         ></readme-content-modal>
+        <quota-modal v-if="quotaModalIsOpen" :cancel="closeQuotaModal" :space="space"></quota-modal>
```

Comment:
You could write this as a self closing tag (same for the readme content modal), which is good practice IMO.